### PR TITLE
RSSFE-40 feat: add leave course mutation

### LINKED ISSUE #125: RSSFE-40 add leave course functionality
Add a button or any other control that will allow a user to leave the course

### PR DESCRIPTION
**Pull request by besovadevka**
Close #125 
Connected new leave course mutation

### CODE REVIEW

File: src/components/Header/components/MenuWrapper/components/CoursesSelect/index.tsx [hunk 1]
```diff
@@ -47,7 +46,7 @@ export const CoursesSelect: FC<CoursesSelectProps> = ({
           {userCourses.length ? <StyledCoursesSelectArrow /> : null}
         </StyledCoursesSelectInfo>
       </StyledCoursesSelectHeaderWrapper>{' '}
-      {userCourses.length ? (
+      {!!userCourses.length ? (
```
Comment (yuliaHope) [inline src/components/Header/components/MenuWrapper/components/CoursesSelect/index.tsx:49]: What is the purpose to use `!!` here?

File: src/hooks/graphql/mutations/useRemoveUserFromCourseMutation.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            courseId,
+            pagination: { skip: page * TEAMS_PER_PAGE, take: TEAMS_PER_PAGE },
+          },
+        });
+
+        const updatedRemovedResults = data?.teams.results
+          .map((team: Team) => {
+            if (team.id === teamId) {
+              if (team.members.length === 1) {
+                return;
```
Comment (yuliaHope) [inline src/hooks/graphql/mutations/useRemoveUserFromCourseMutation.ts:35]: `return null` will be more clear

File: src/modules/TeamsList/components/TeamListModals/index.tsx [hunk 3]
```diff
@@ -136,6 +154,21 @@ export const TeamListModals: FC<{ page: number }> = ({ page }) => {
     expelUserFromTeam();
   };
 
+  const onSubmitRemoveCourseModal = () => {
+    removeUserFromCourse().then(({ data: { removeUserFromCourse } }) => {
+      dispatch({
+        type: SET_USER_DATA,
+        payload: removeUserFromCourse,
+      });
+      if (!!removeUserFromCourse.courses.length) {
```
Comment (yuliaHope) [inline src/modules/TeamsList/components/TeamListModals/index.tsx:163]: the same here

**Review by yuliaHope — APPROVED** [review]

File: src/modules/TeamsList/components/Teams/components/MyTeam/index.tsx [hunk 4]
```diff
@@ -24,6 +24,9 @@ export const MyTeam: FC<MyTeamProps> = ({ team, userId }) => {
   const dispatch = useDispatch();
 
   const leaveTeam = () => dispatch({ type: ACTIVE_MODAL_LEAVE, payload: true });
+  const removeCourse = () =>
+    dispatch({ type: ACTIVE_MODAL_REMOVE_COURSE, payload: true });
+  // const buttonsInfo: [string, () => void] = [];
```
Comment (MadaShindeInai) [inline src/modules/TeamsList/components/Teams/components/MyTeam/index.tsx:29]: remove commented code

**Review by MadaShindeInai — APPROVED** [review]

**Pull request merged**